packages/bpk-component-content-cards/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bpk-component-content-cards",
+  "version": "0.0.1",
+  "description": "Backpack content-cards component.",
+  "license": "Apache-2.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "bpk-mixins": "^31.1.6",
+    "prop-types": "^15.5.8"
```

Comment:
Is this used?

Reply:
`bpk-mixins` is used but not `prop-types`, will remove it 👍